pom.xml

```diff
@@ -81,24 +81,49 @@
         <!-- No need to build the javadocs per module. Aggregated javadocs are build in the distribution module. See also HV-894 -->
         <maven.javadoc.skip>true</maven.javadoc.skip>
 
-        <!-- Don't upgrade chronos to 1.1.0 yet. Dependencies get copied to wrong location -->
-        <org.codehaus.mojo.chronos.version>1.0-beta-3</org.codehaus.mojo.chronos.version>
         <bv.api.version>2.0.0-SNAPSHOT</bv.api.version>
-        <classmate.version>1.1.0</classmate.version>
-        <paranamer.version>2.5.5</paranamer.version>
-        <joda-time.version>2.7</joda-time.version>
-        <jsoup.version>1.8.3</jsoup.version>
-        <wildfly.version>10.0.0.Final</wildfly.version>
-        <arquillian.version>1.1.9.Final</arquillian.version>
-        <slf4j.version>1.7.2</slf4j.version>
+
+        <paranamer.version>2.8</paranamer.version>
+        <javax.el.version>3.0.1-b08</javax.el.version>
         <jboss.logging.version>3.3.0.Final</jboss.logging.version>
-        <javax.el.version>2.2.4</javax.el.version>
         <jboss.logging.processor.version>2.0.1.Final</jboss.logging.processor.version>
 
-        <!-- Used to add further arguments to the arg line for specific profiles -->
-        <surefire.argLine.extension></surefire.argLine.extension>
+        <!-- Don't upgrade chronos to 1.1.0 yet. Dependencies get copied to wrong location -->
+        <org.codehaus.mojo.chronos.version>1.0-beta-3</org.codehaus.mojo.chronos.version>
+
+        <!--
+            These dependencies should be aligned with the ones from the Wildfly version we support
+            See http://search.maven.org/#search|gav|1|g%3A"org.wildfly"%20AND%20a%3A"wildfly-parent"
+        -->
+        <classmate.version>1.3.1</classmate.version>
```

Comment:
Is this the one used in WF (10.1)? Same question for the other deps just in case.